### DISCUSSION TITLE
Can't Play Music but I have a YouTube API Key

### DISCUSSION THREAD

**Discussion by MRSgami** [eritislami/evobot#844] [General]
So I created a panel for my MC server and I decided to make a music bot for my discord (i put the files on the panel) and turned the bot on. But no music is playing and I have a YouTube API key for it _(Its Pterodactyl Panel)_

And if you know how to fix DM me on Discord: Gami#5963

**Comment by adamdev-id** [comment]
Instead of using 
```
message.reply("message");
```
use
```
message.channel.send("message");
```
